w3af/core/data/request/fuzzable_request.py

```diff
@@ -429,6 +434,29 @@ def set_headers(self, headers):
 
         self._headers = headers
 
+    def set_force_fuzzing_headers(self, headers):
+        """
+        TODO
+        :param headers:
+        :return:
+        """
+        if headers is None:
+            raise TypeError('headers should not be null')
+
+        if not isinstance(headers, collections.Iterable):
+            raise TypeError(TYPE_ERROR % ('_force_fuzzing_headers', 'iterable'))
+
+        self._force_fuzzing_headers = set()
+        for header in headers:
+            self._force_fuzzing_headers.add(header)
```

Comment:
Why like this and not: `self._force_fuzzing_headers = set(headers)`?

If you do it like this, it is not possible for someone to remove a header from the set once it has been set.

Reply:
`self._force_fuzzing_headers = set(headers)` looks better - I just overcomplicated this :) I'll update it. Thanks!